modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/ToolsEndpoint.java

```diff
@@ -687,6 +688,9 @@ public Response editVideo(@PathParam("mediapackageid") final String mediaPackage
         } catch (NotFoundException e) {
           logger.warn("Workflow '{}' not found", workflowId);
           return R.badRequest("Workflow not found");
+        } catch (UnauthorizedException e) {
+          logger.warn("Workflow '{}' not accessible", workflowId);
```

Comment:
I don't think we need to log this here, a user could flood the log otherwise by trying to access a workflow they're not authorized for.